CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ Mapbox welcomes participation and contributions from everyone.
 
 ### v0.41.0 - June 26, 2019
 
+* Fix DynamicCamera#CameraPosition.zoom NPE [#1979](https://github.com/mapbox/mapbox-navigation-android/pull/1979)
```

Comment:
btw, why are you doing this with every PR?

Reply:
So we don't need to go over all the PRs that got merged and include them in the `CHANGELOG` pre-release i.e. long story short to improve the release process 😉

We used to have a checkbox in the release checklist to do this 👀 https://github.com/mapbox/mapbox-navigation-android/issues/1920 and it was a hard task that made the release process slower so we thought that could be done incrementally on every PR, making the process more straightforward. This was a quick improvement that didn't involve any extra automation, ideally this should be autogenerated as part of the release build.